### ISSUE TITLE
A11y_code.visualstudio.com_Docs Node.JS/Javascript_Node.js Debugging_ScreenReader: Screen reader is not announcing proper information about input instruction(Limit).

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#2819]
“[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details**:
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 3.3.2

**Repro Steps**:
Navigate to the URL: [https://code.visualstudio.com/docs/nodejs/nodejs-debugging](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)
follow below steps:
1.Launch the application URL: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "Docs" link.
3.From left navigation, select "Node.JS/Javascript".
4.Navigate to "Node.js Debugging" link.
5.Start the screen reader.
6.Navigate to feedback section and click on "Yes" feedback button.

**Actual**:
Issue1: When we hit "Yes" button, Additional Feedback section is opened and announced but screen reader does not announce anything about Input limit "1000". Even it is not specified that 1000 words or characters.

Issue2: When we type something in Additional Feedback edit section, the count of input limit get decreased but screen reader does not announce anything about it.   

![image](https://user-images.githubusercontent.com/52035395/60169698-d1589980-9824-11e9-9d72-cad2be9329c3.png)

This issue is observed with Edge+Narrator and Chrome+NVDA

**Expected**:
Issue1: Screen reader should announce that 1000 is input character limit along with "Advance Feedback" section.
Issue2: Screen reader should announce dynamic change in input limit when we type feedback.

**Recommendations**:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact**:
Issue1: Screen reader user is unable to understand about input limit. 
Issue1: Screen reader user is unable to understand remaining input limit.

**MAS Reference**:
MAS 3.3.2 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf})

**Attachment for Reference**:
[Code.visualstudio.com_Node.js Debugging_Input instruction.zip](https://github.com/microsoft/vscode-docs/files/3329634/Code.visualstudio.com_Node.js.Debugging_Input.instruction.zip)

**Comment by RohitJadhav-TCS** [comment]
Similar issue is observed in through out application where the Feedback section is present.
